From 596d95aef2edf1fe8b79d78b8e1a05a96cad5ef2 Mon Sep 17 00:00:00 2001
From: Aedan McHale <aedan.mchale@codethink.co.uk>
Date: Mon, 19 Oct 2026 10:07:22 +0100
Subject: [PATCH] [PATCH] Drop pybind11

This is only used for parallel compilation of the C extension but
pybind11 depends on a bunch of things like boost, scikit etc. which
makes it annoying to package, so drop this.
---
 pyproject.toml | 1 -
 setup.py       | 2 --
 2 files changed, 3 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index d978f3943..65fb844ac 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,7 +1,6 @@
 [build-system]
 build-backend = "backend"
 requires = [
-  "pybind11",
   "setuptools>=77",
 ]
 backend-path = [
diff --git a/setup.py b/setup.py
index 2d02cf129..6a4a749ac 100644
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,6 @@ import sys
 import sysconfig
 import warnings
 
-from pybind11.setup_helpers import ParallelCompile
 from setuptools import Extension, setup
 from setuptools.command.build_ext import build_ext
 
@@ -35,7 +34,6 @@ while sys.argv[-1].startswith("--pillow-configuration="):
     configuration.setdefault(key, []).append(value)
 
 parallel_default = int(configuration.get("parallel", ["0"])[-1])
-ParallelCompile("MAX_CONCURRENCY", parallel_default).install()
 
 
 def get_version() -> str:
-- 
2.55.0

